app/models/startup.rb

```diff
@@ -27,7 +27,6 @@ class Startup < ApplicationRecord
   has_one :school, through: :course
 
   validates :name, presence: true
-  validates :level, presence: true
```

Comment:
This looks like an unrelated change.

Reply:
Rails threw an error as `  belongs_to :level` ensures this validation by default and this is redundant